financialaid/models_test.py

```diff
@@ -90,3 +87,25 @@ def test_save_and_log(self):  # pylint: disable=no-self-use
                 self.assertAlmostEqual(value, original_after_json[field])
             else:
                 assert value == original_after_json[field]
+
+    def test_to_dict(self):
+        """
+        Tests that to_dict() function on FinancialAid
+        """
+        financial_aid = FinancialAidFactory.create()
+        financial_aid_dict = financial_aid.to_dict()
+        assert financial_aid_dict["user"] == financial_aid.user.id
+        assert financial_aid_dict["tier_program"] == financial_aid.tier_program.id
+        assert financial_aid_dict["status"] == financial_aid.status
+        assert financial_aid_dict["original_currency"] == financial_aid.original_currency
+        assert financial_aid_dict["country_of_income"] == financial_aid.country_of_income
+        self.assertAlmostEqual(financial_aid_dict["income_usd"], financial_aid.income_usd)
+        self.assertAlmostEqual(financial_aid_dict["original_income"], financial_aid.original_income)
+        if financial_aid.date_exchange_rate is None:
+            assert financial_aid_dict["date_exchange_rate"] == financial_aid.date_exchange_rate
+        else:
+            assert financial_aid_dict["date_exchange_rate"] == str(financial_aid.date_exchange_rate)
+        if financial_aid.date_documents_sent is None:
+            assert financial_aid_dict["date_documents_sent"] == financial_aid.date_documents_sent
+        else:
+            assert financial_aid_dict["date_documents_sent"] == str(financial_aid.date_documents_sent)
```

Comment:
I think you need multiple tests to test each case and not have an `if` statement in the test.